src/puppetlabs/trapperkeeper/authorization/rules.clj

```diff
@@ -119,11 +123,20 @@
   [a b]
   (or (= a b) (some #{:any} [a b])))
 
+(defn- query-params-match?
```

Comment:
I tend to use "private" methods to bootstrap the middle of a function call chain for testing. It's possible this may not be the best way to test something, but it's how I tend to roll. In general, when making a library I would generally only expose "public" functions from a core namespace, and stuff all the internals somewhere else. In this case, the "API" level stuff is in the service definition.